### ISSUE TITLE
Warning: slice start greater than slice end

### ISSUE THREAD

**Issue by sigven** [konradjk/loftee#31]
Hi,

I am getting a warning for some variants related to slicing, which disables the plugin from working. Could this be related to strange input variants, or is this a potential bug? Unfortunately, the actual variants associated with the warnings were not reported by VEP, so I do not have a good idea of the places in which the warning hits in.

Excerpt from VEP's warning output below:

_____

WARNING: Plugin 'LoF' went wrong: 
-------------------- EXCEPTION --------------------
MSG: Slice start cannot be greater than slice end
STACK Bio::EnsEMBL::Slice::expand /home/vep/src/ensembl-vep/Bio/EnsEMBL/Slice.pm:1247
STACK LoF::get_upstream_acceptor_flank /home/vep/src/ensembl-vep/modules/extended_splice.pl:209
STACK LoF::get_effect_on_splice /home/vep/src/ensembl-vep/modules/extended_splice.pl:111
STACK LoF::run /home/vep/src/ensembl-vep/modules/LoF.pm:180
STACK (eval) /home/vep/src/ensembl-vep/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:1898

Date (localtime)    = Thu Jul  5 12:19:01 2018
Ensembl API version = 92
_____

regards,
Sigve

**Comment by konradjk** [comment]
Yeah, I suspect this is due to newer version of VEP and the cache that LOFTEE is used to. We've tested it up to v85 (beyond which I believe there are only marginal improvements for GRCh37). Are you running on GRCh37 or 38?

**Comment by sigven** [comment]
I am serving both GRCh37 and GRCh38, now using the most recent v95 of the API. I will have a a check if it still returns the same warnings.

**Comment by wavefancy** [comment]
I have the same error MSG for GRCh37 by using Ensembl API 96. Any luck with this? Thanks much!

Best regards
Wallace

**Comment by christopherdkoch** [comment]
Has anyone been able to resolve this issue? I'm seeing this as well with the most recent version of VEP and GRCh37.

**Comment by Frogdong** [comment]
I have the same error MSG for GRCh38 by using Ensembl API 101, The warning as follows:
-------------------- EXCEPTION --------------------
MSG: Slice start cannot be greater than slice end
STACK Bio::EnsEMBL::Slice::expand /home/tools/ensembl-vep-release-101/Bio/EnsEMBL/Slice.pm:1217
STACK LoF::get_upstream_acceptor_flank /home/annotation/VEP/LOFTEE/loftee/extended_splice.pl:209
STACK LoF::get_effect_on_splice /home/annotation/VEP/LOFTEE/loftee/extended_splice.pl:111
STACK LoF::run /home/annotation/VEP/LOFTEE/loftee/LoF.pm:188
STACK (eval) /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:2042
STACK Bio::EnsEMBL::VEP::OutputFactory::run_plugins/home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:2041
STACK Bio::EnsEMBL::VEP::OutputFactory::get_all_VariationFeatureOverlapAllele_output_hashes /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:387
STACK Bio::EnsEMBL::VEP::OutputFactory::get_all_output_hashes_by_VariationFeature /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:337
STACK Bio::EnsEMBL::VEP::OutputFactory::VCF::get_all_lines_by_InputBuffer /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory/VCF.pm:313
STACK Bio::EnsEMBL::VEP::Runner::_buffer_to_output /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:425
STACK Bio::EnsEMBL::VEP::Runner::next_output_line /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:373
STACK Bio::EnsEMBL::VEP::Runner::run /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:211
STACK toplevel /home/tools/bin/vep:228
Date (localtime)    = Thu Jul 22 00:07:40 2021
Ensembl API version = 101
---------------------------------------------------
Has anyone solved this problem? many thanks!

**Comment by Frogdong** [comment]
I have the same error MSG for GRCh38 by using Ensembl API 101, The warning as follows:
-------------------- EXCEPTION --------------------
MSG: Slice start cannot be greater than slice end
STACK Bio::EnsEMBL::Slice::expand /home/tools/ensembl-vep-release-101/Bio/EnsEMBL/Slice.pm:1217
STACK LoF::get_upstream_acceptor_flank /home/annotation/VEP/LOFTEE/loftee/extended_splice.pl:209
STACK LoF::get_effect_on_splice /home/annotation/VEP/LOFTEE/loftee/extended_splice.pl:111
STACK LoF::run /home/annotation/VEP/LOFTEE/loftee/LoF.pm:188
STACK (eval) /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:2042
STACK Bio::EnsEMBL::VEP::OutputFactory::run_plugins/home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:2041
STACK Bio::EnsEMBL::VEP::OutputFactory::get_all_VariationFeatureOverlapAllele_output_hashes /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:387
STACK Bio::EnsEMBL::VEP::OutputFactory::get_all_output_hashes_by_VariationFeature /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:337
STACK Bio::EnsEMBL::VEP::OutputFactory::VCF::get_all_lines_by_InputBuffer /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory/VCF.pm:313
STACK Bio::EnsEMBL::VEP::Runner::_buffer_to_output /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:425
STACK Bio::EnsEMBL::VEP::Runner::next_output_line /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:373
STACK Bio::EnsEMBL::VEP::Runner::run /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:211
STACK toplevel /home/tools/bin/vep:228
Date (localtime) = Thu Jul 22 00:07:40 2021
Ensembl API version = 101
--------------------------------------------------------------------
Has anyone solved this problem? many thanks!

**Comment by Frogdong** [comment]
I have the same error MSG for GRCh38 by using Ensembl API 101
-------------------- EXCEPTION --------------------
MSG: Slice start cannot be greater than slice end
STACK Bio::EnsEMBL::Slice::expand /home/tools/ensembl-vep-release-101/Bio/EnsEMBL/Slice.pm:1217
STACK LoF::get_upstream_acceptor_flank /home/annotation/VEP/LOFTEE/loftee/extended_splice.pl:209
STACK LoF::get_effect_on_splice /home/annotation/VEP/LOFTEE/loftee/extended_splice.pl:111
STACK LoF::run /home/annotation/VEP/LOFTEE/loftee/LoF.pm:188
STACK (eval) /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:2042
STACK Bio::EnsEMBL::VEP::OutputFactory::run_plugins/home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:2041
STACK Bio::EnsEMBL::VEP::OutputFactory::get_all_VariationFeatureOverlapAllele_output_hashes /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:387
STACK Bio::EnsEMBL::VEP::OutputFactory::get_all_output_hashes_by_VariationFeature /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory.pm:337
STACK Bio::EnsEMBL::VEP::OutputFactory::VCF::get_all_lines_by_InputBuffer /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/OutputFactory/VCF.pm:313
STACK Bio::EnsEMBL::VEP::Runner::_buffer_to_output /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:425
STACK Bio::EnsEMBL::VEP::Runner::next_output_line /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:373
STACK Bio::EnsEMBL::VEP::Runner::run /home/tools/ensembl-vep-release-101/modules/Bio/EnsEMBL/VEP/Runner.pm:211
STACK toplevel /home/tools/bin/vep:228
Date (localtime)    = Thu Jul 22 00:07:40 2021
Ensembl API version = 101
---------------------------------------------------
Has anyone solved this problem? Many thanks!

**Comment by greener98103** [comment]
I'm seeing the same error. Any resolution to this? Thanks
